browser/resources/settings/brave_settings_overrides.js

```diff
@@ -354,18 +354,13 @@ BravePatching.RegisterPolymerTemplateModifications({
     // (we remove the People section as a separate section).
     const page = templateContent.querySelector('settings-animated-pages[section=people]')
     page.setAttribute('section', 'getStarted')
-    const profileTemplate = templateContent.querySelector('template[is="dom-if"][if="[[diceEnabled_]]"]')
-    // In chromium, the whole 'manage profile' section is only
-    // displayed if dice is enabled.
-    // Instead, always show it, but remove the google account specific entry.
-    profileTemplate.setAttribute('if', 'true')
-    const manageGoogleAccount = profileTemplate.content.querySelector('#manage-google-account')
+    const manageGoogleAccount = templateContent.querySelector('#manage-google-account')
```

Comment:
Had to look up what DICE is; `Desktop Identity Consistency`, meaning the sign in for the browser also applies to the cookies for GAIA services. We don't use this of course, but was interesting to see that this is enabled completely now for 100% of Chromium/Chrome users